addons/godot-firebase/containers/circular_container/circular_container.gd

```diff
@@ -0,0 +1,44 @@
+tool
```

Comment:
Didn't we remove these files from here and put them all in GodotFirebase-UI?

Reply:
Yep, I don't know why they are back honestly